src/main/kotlin/InputVerificator.kt

```diff
@@ -0,0 +1,40 @@
+import java.util.Scanner
+
+class InputVerificator {
+    companion object{
+        private val currentScanner = Scanner(System.`in`)
```

Comment:
Чтобы не писать весь код внутри `companion object` класса, ты можешь сам класс сделать объектом:
```suggestion
object InputVerificator {
        private val currentScanner = Scanner(System.`in`)
```